src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -2838,24 +2838,6 @@ HashableClauseMutator(Node *originalNode, Var *partitionColumn)
 			newNode = (Node *) hashedOperatorExpression;
 		}
 	}
-	else if (IsA(originalNode, NullTest))
-	{
-		NullTest *nullTest = (NullTest *) originalNode;
-		Var *column = NULL;
-
-		Expr *nullTestOperand = nullTest->arg;
-		if (IsA(nullTestOperand, Var))
-		{
-			column = (Var *) nullTestOperand;
-		}
-
-		if ((column != NULL) && equal(column, partitionColumn) &&
-			(nullTest->nulltesttype == IS_NULL))
-		{
-			OpExpr *opExpressionWithZeroConst = MakeOpExpressionWithZeroConst();
```

Comment:
Yeah, the compiler probably warned about it because when I opened it up in Eclipse it was highlighted as unused. I removed it (and its declaration) and pushed.